### PR TITLE
navbar

### PR DESCRIPTION
**Pull request by SumitSaundadkar**
NavBar

### CODE REVIEW

**Review by TusharJade — COMMENTED** [review]
Everything looks great, code is clean good to go for merge. Just nav bar links are not working like alert, badge are not working when clicked you can fix that

File: components/navigation.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
+.nav_items{
```
Comment (aiman-ansari) [inline components/navigation.css]: use ( - )  instead of underscore in your class names

**Review by aiman-ansari — CHANGES_REQUESTED** [review]

**Review by m26unkwn — APPROVED** [review]
everything is good to go.
 Things you can update

1. Instead of using default color you can use different color for button text.

File: components/navigation.css (hunk 1 above)
Comment (SumitSaundadkar) [reply components/navigation.css]: done thank you.

**Pull request merged**